README.md

```diff
@@ -102,6 +102,8 @@ Install dependencies:
 npm install orbit-db ipfs
 ```
 
+***NOTE!*** Compatible with **js-ipfs versions <= 0.46**
```

Comment:
Sorry! I should have been more clear. I liked where this was before but also commit my suggestion as well :)

Reply:
oh right!! on it